### ISSUE TITLE
Remove 'singularity' with 'apptainer' everywhere

### ISSUE THREAD

**Issue by klieret** [hsf-training/hsf-training-singularity-webpage#108]

**Comment by mambelli** [comment]
Make "apptainer" the primary software:
- replace singularity w/ apptainer in the commands
- use the Apptainer name first

Should also the training name/link be changed to "[hsf-training-apptainer-webpage](https://github.com/hsf-training/hsf-training-apptainer-webpage)" ?

**Comment by klieret** [comment]
> Should also the training name/link be changed to "[hsf-training-apptainer-webpage](https://github.com/hsf-training/hsf-training-apptainer-webpage)" ?

The only bad thing about this is that it's gonna break the links to the rendered webpage. But this problem keeps on coming up and I also don't like the addresses in the first place. 

I actually think that it would be much cooler to have some DNS stuff to get some nice addresses in the first place.  Like hepsoftwarefoundation.org/apptainer-training etc.
I actually really like the idea to just buy `hsftraining.org`, but I remember that it's quite a hazzle to have permanent funds allocated for something like that.

**Comment by mambelli** [comment]
And the training names are not all consistent. All start with "hsf-training-" some end in "-webpage", some not (e.g. docker)

**Comment by klieret** [comment]
I know...

**Comment by GuillermoFidalgo** [comment]
I have not yet looked at the reason why it would break all the links but I would naively suggest to use the liquid tag `{% link %}`. This should ensure that all links work as long as we point correctly along with the usage of `baseurl` or the `url` filters.
I believe these can be set in the `config.yml` file.


Disregard this comment if it doesn't really help in our situation.
Comment also relevant for https://github.com/hsf-training/hsf-training-cicd/issues/41

**Comment by klieret** [comment]
no, I just meant the links to the website (from somewhere else) :) 

The internal links should all survive indeed :)

**Comment by mambelli** [comment]
There were some hardcoded links in the instances episode. I fixed them already using `{{ page.root }}{% link _episodes/05-definition-files.md %}`.  Let me know if there is a better way. I did not change the config assuming that is all OK.

I agree w/ @klieret about the external links. It should be OK after an adjustment period. Something to think moving forward is a uniform naming convention (some training end in `-webpage`, some do not)
